packages/api/src/router/review.ts

```diff
@@ -72,7 +73,38 @@ export const reviewRouter = {
 
   create: protectedProcedure
     .input(CreateReviewSchema)
-    .mutation(({ ctx, input }) => {
+    .mutation(async ({ ctx, input }) => {
+      console.log("new review");
+      if (!input.profileId) {
+        console.log("no profile id");
+        throw new TRPCError({
+          code: "FORBIDDEN",
```

Comment:
PRECONDITION_FAILED could be another tRPC error we can use but it really is just up to whatever you feel lol

https://trpc.io/docs/server/error-handling